lib/datadog/appsec/contrib/rails/ext.rb

```diff
@@ -1,3 +1,4 @@
+# typed: true
```

Comment:
Should comment directives have an empty line separating them from the `module`/`class` items? Presumably having these non-separated makes the comment part of the module or class docstring.